vtrace/breakpoints.py

```diff
@@ -404,3 +411,17 @@ def notify(self, event, trace):
         ret_addr, args = tup
 
         self.runPostHookCallbacks(event, trace, ret_addr, args)
+
+class PosixLibLoadHookBreakpoint(Breakpoint):
+    '''
+    POSIX systems need to hook DL to identfy when libraries are loaded.
+    '''
+    def __init__(self, expression):
+        Breakpoint.__init__(self, None, expression=expression)
+        self.untouchable = True
+        self.silent = True
+
+    def notify(self, event, trace):
+        logger.debug("PosixLibLoadHookBreakpoint: reanalyze maps and resolve symbols")
+        trace._findLibraryMaps(b'\x7fELF', always=True)
+        # handle unresolved expression bp's 
```

Comment:
specialized breakpoint just for handling POSIX library events, loading the new library/maps and handling bp resolution.  this is the core of replicating Windows' LOAD_LIBRARY behavior on POSIX/Linux